community/kernel/src/main/java/org/neo4j/kernel/impl/store/StoreFactory.java

```diff
@@ -148,12 +148,6 @@ public NeoStore newNeoStore( boolean allowCreateEmpty )
             return createNeoStore();
         }
 
-        CountsTracker countsTracker = null;
-        if ( storeExists && countsStoreExists() )
-        {
-            countsTracker = newCountsStore();
-        }
-
```

Comment:
But you call `init()` in the `NeoStore` constructor, i.e., immediately after this. No?